docs/template_extension/new/CMakeLists.txt

```diff
@@ -22,7 +22,3 @@ if(OpenVINO_Frontend_ONNX_FOUND)
     target_compile_definitions(${TARGET_NAME} PRIVATE OPENVINO_ONNX_FRONTEND_ENABLED)
 endif()
 # [cmake:extension]
-
-# Enable code style check
-file(GLOB_RECURSE template_extension_src "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp" "${CMAKE_CURRENT_SOURCE_DIR}/*.hpp")
-add_clang_format_target(${TARGET_NAME}_clang FOR_SOURCES ${template_extension_src})
```

Comment:
BTW, we can just keep as is, but use `if(COMMAND add_clang_format_target)`

Reply:
I just wanted to remove redundant commands from cmake example.